blockcerts/README.md

```diff
@@ -3,12 +3,12 @@
 
 Homepage:
 
-https://w3id.org/blockcerts
+http://w3id.org/blockcerts
```

Comment:
Leave this as https.  w3id.org redirects all http traffic to https so you should avoid the extra redirect.